WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -180,6 +183,8 @@ enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
             return "wordpress_support_forum_remote_field"
         case .blaze:
             return "blaze"
+        case .jetpackMigrationSupportCard:
+            return "jetpack_migration_faq_card_remote_field"
```

Comment:
What's your take on the remote feature flag? 🤔

I wonder how cautious we want to be with such a feature. Theoretically, we could remove the feature from the code when we think it's no longer necessary and not rely on the remote feature flag.

Reply:
I see this is removed now so not sure if this question is still open. I think they add most value when features need to be rolled out on a specific date after release or may need to be reverted remotely. Also, I saw some work today on the backend which enables a gradual rollout of a feature flag, which is another use case.
Here, it doesn't seem like any of those scenarios, so not using a feature flag makes sense.

Reply:
Yes, I still wanted to here your opinion, thanks.

We exchanged messages with @ravishanker about that and decided not to use it as well.